### PR TITLE
OCL-4: Project Initialization And SetUp

### PR DESCRIPTION
**Pull request by EleisonC**
# JIRA TICKET NAME:
[OCL-4: Project Initialization And SetUp](https://issues.openmrs.org/browse/OCL-4)

# Summary:
This contains the initial setup of the project with the development environment and a Readme

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you squash the commits into one as advised at? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by mwaz** [comment]
Eslint would come in quite handy if added during setup.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+### Installation Process
+```
+- Fork the repository https://github.com/openmrs/openmrs-ocl-client#fork-destination-box
+- Clone the forked repository to your computer
+    # git clone [the url of the repository]
+- Run 
+    $ npm install(if using npm)
+    $ yarn install (if using yarn)
+```
+### Runnig tests
```
Comment (shakirandagire) [inline README.md]: There is a typo on Running tests. Could you rectify it.

Comment (EleisonC) [reply README.md]: On it

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb"
+}
```
Comment (Betty-Kebenei) [inline .eslintrc.json]: EOF error here.

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "short_name": "openmrs-ocl-client",
+  "name": "openmrs-ocl-client",
+  "start_url": "./index.html",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```
Comment (Betty-Kebenei) [inline public/manifest.json]: EOF error here too

File: .eslintrc.json (hunk 2 above)
Comment (hadijahkyampeire) [reply .eslintrc.json]: @Betty-Kebenei Yah I know the format I was just suggesting a template to ease the PR process

Comment (EleisonC) [reply .eslintrc.json]: I have worked on that

**Comment by EleisonC** [comment]
@Betty-Kebenei I took what you said into consideration talked to my team and got rif of the `package-lock.json`

**Comment by EleisonC** [comment]
@shakirandagire I corrected the typo that you spotted thank you very much for that

**Comment by EleisonC** [comment]
@hadijahkyampeire I was able to generate A template for PRs as you requested

**Comment by EleisonC** [comment]
@mwaz I took your feedback into consideration and and set up Eslint

**Comment by EleisonC** [comment]
@dkayiwa Yes I did follow your advice and thank you

File: public/manifest.json (hunk 3 above)
Comment (EleisonC) [reply public/manifest.json]: I have worked on that to

File: README.md [hunk 4]
```diff
@@ -1,2 +1,39 @@
...
+> allow reusability of the existing concepts and allow one to create their concepts if they are not in exsitance.
+
+## Build
+### Requirements
+
+- Node and Npm [Here](https://nodejs.org/en/download/).
+
+### Installation Process
+```
+- Fork the repository https://github.com/openmrs/openmrs-ocl-client#fork-destination-box
```
Comment (dkayiwa) [inline README.md]: What url should one fork?

Comment (EleisonC) [reply README.md]: I copied the URL we are redirected to when we click the fork button but I have discovered that the only alternative is to click the fork button let me rectify that

**Pull request merged**